setup/functions/auth.js

```diff
@@ -48,7 +48,8 @@ export default (request, response) => {
             return db.set(key, record, 525600).then(() => {
                 return allow();
             }).catch((err) => {
-                return die(err);
```

Comment:
this only occurs when there is an error with the KVStore, so we want to tell the client "Internal server error" not precisely what happened on the back end